spec/system/jobseekers/jobseekers_can_delete_a_draft_job_application_spec.rb

```diff
@@ -11,6 +11,7 @@
   after { logout }
 
   it "allows deleting the draft permanently" do
+    expect(JobApplication.count).to eq 1
```

Comment:
Later in the test we assert that JobApplication.count is zero, so adding this initial assertion to ensure that the test is working as expected.